dm/master/server.go

```diff
@@ -246,14 +246,9 @@ func (s *Server) StartTask(ctx context.Context, req *pb.StartTaskRequest) (*pb.S
 				return
 			}
 			workerResp, err := cli.StartSubTask(ctx, &pb.StartSubTaskRequest{Task: stCfgToml})
-			if err != nil {
-				workerResp = &pb.CommonWorkerResponse{
-					Result: false,
-					Msg:    errors.ErrorStack(err),
-				}
-			}
-			workerResp.Worker = worker
-			workerRespCh <- workerResp
+			workerResp = s.handleOperationResult(ctx, cli, stCfg.Name, workerResp.LogID, err, workerResp)
```

Comment:
if `err == nil`, `workerResp.LogID` may panic? only pass `workerResp` is enough?